app/controllers/trips_controller.rb

```diff
@@ -77,6 +81,28 @@ def destroy
   def download
   end
 
+  def reject
+    if (can? :read, @trip) && @trip.status == 'applied'
+      @trip.update_attribute(:status, 'declined')
+      @trip.update_attribute(:last_modified, Date.today)
+      redirect_to @trip.user
+    else
+      redirect_to root_path
+      flash[:error] = t('trip.not_authorized')
+    end
```

Comment:
Das wollte ich eig schon in der Beschreibung des Requests fragen: Der Teil wird durch die before_action Zeile am Beginn des Controllers wohl nie erreicht. Daher könnte man den auch entfernen oder?